exporter/clickhouseexporter/config.go

```diff
@@ -134,5 +145,26 @@ func (cfg *Config) buildDB(database string) (*sql.DB, error) {
 	}
 
 	return conn, nil
+}
+
+// TableEngineString generates the ENGINE string.
+func (cfg *Config) TableEngineString() string {
+	engine := cfg.TableEngine.Name
+	params := cfg.TableEngine.Params
+
+	if cfg.TableEngine.Name == "" {
+		engine = defaultTableEngineName
+		params = ""
```

Comment:
I think the params should be checked independently, there is a case:
```
clickhouse/a-possible-engine-params:
  endpoint: clickhouse://127.0.0.1:9000
  table_engine:
    params: "'/clickhouse/tables/{shard}/table_name', '{replica}', ver"
```

result in `MergeTree('/clickhouse/tables/{shard}/table_name', '{replica}', ver)`

Reply:
I thought about this, and I think they should provide the table name specifically instead of relying on the default. I could add an error message for providing `params` without `name` to further enforce this.

Also, [merge tree does not have any parameters](https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/mergetree#engine)